docs/specs/mlnet-cli/MLNET-CLI-Specs.md

```diff
@@ -0,0 +1,969 @@
+    `ludwig visualize --visualization compare_performance --test_statistics path/to/test_statistics_model_1.json path/to/test_statistics_model_2.json`
+
+# Open questions
+
+- If there's a 1:1 mapping between the first level of paramenters in the .YAML file and the CLI arguments, how do we deal with expected syntax/naming of the args? Same IDs/text?: 
+    - `--test-dataset` vs. `test_dataset`
+
+- If we were using a sub-command for the ML-Task (such as `mlnet new regression`) instead of a regular argument (such as `mlnet new --ml-task regression`), how can a sub-command be specified in the response file?
+
+    How do you specify a sub-command?
+Like the first line below?
+
+```
+# Response File (.rsp) with arguments for the ML.NET CLI
+
+/BinaryClassification
+/--dataset: "/MyExperiments/MyDataSets/Population-Training.csv"
+/--test-dataset: "/MyExperiments/MyDataSets/Population-Test.csv"
+/--label-column-name: InsuranceRisk
+/--model-settings-file: "./my_model_settings.yaml"
+```
+
+    - How do we know what sub-command is in the .RSP file it if we had several sub-commands? Based on the order?
+
+
+- The "best models" number to return is configurable. However, what will be the maximum number of "best models" that can be returned? Could be 10? What's the maximum number of "best models" that AutoML API can return? (To confirm this maximum number).
+
+- Should we use the term 'column', 'field', 'variable' when meaning the input columns or the objective dependent variable?
+--> In ML.NET we usually use the 'columns' term..
+--> Gleb's: Note that there is a duality of "the source file column" and the "output column". Several input columns can turn into a single vector IDV column. All that being said, we should use same terms as in excel or sql data import - which is column, I believe.
+--> Cesar: Let's use column.
+
+- For the debug-log-file option, what's the file extenssion (.log ?) and internal format of the file? - Confirm this.
+
+- Will overriden default values just be provided as static values in the CLI commands or also based on a related configuration .JSON file placed along with the CLI executable? See [comparable .JSON files for dotnet CLI templates](https://github.com/dotnet/dotnet-template-samples/blob/master/05-multi-project/.template.config/template.json))
+
+- Support Custom templates for "automlnet new" such as [Custom templates for dotnet new](https://docs.microsoft.com/en-us/dotnet/core/tools/custom-templates)? - That could allow extensibility for other application project types or even for other languages like F# or additional scenarios.
+CESAR --> No, the CLI team confirmed that custom templates were meant to be created by static code, not for code generation. It would be too complex.
+
+
+
+
+## Additional characteristics to be defined
+
+- Stopping criteria  - { Default timeout or timeout provided by the user }
+
+Gleb's (Cesar: Although, ins't this related to AutoML API instead the CLI?):
```

Comment:
I don't understand this and the below "priority order" list.

Is this just an author's "TODO Notes" list?

(nit) `ins't`.